Dockerfile

```diff
@@ -1,7 +1,6 @@
 FROM alpine:latest
 
-RUN apk -Uuv \
-    add \
+RUN apk add --no-cache \
```

Comment:
I couldn't find `-u` in the docs, and I didn't think the `-v` was useful. Would be glad to add either of those back.